spec/views/categories_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+describe 'User category page', type: :feature do
+  before :each do
+    login_user
+  end
+
+  it 'I can see  Categories of user' do
+    visit '/categories'
+    expect(page).to have_content('Categories')
+  end
+
+  it 'The category page should have div anchor tags to take you to the transaction page of that category' do
+    visit '/categories'
```

Comment:
Please find below the error message received when I tried to create a `category`:

![image](https://user-images.githubusercontent.com/10332499/170846578-d6ff4d2c-a002-414a-91f7-184f78c04df2.png)

![image](https://user-images.githubusercontent.com/10332499/170846584-c6925983-383b-46d9-979c-c730323ba9cb.png)

Because I cannot create a category, `I am unable to check other requirements.` Kindly fix this error.